x/cronos/spec/01_concepts.md

```diff
@@ -0,0 +1,45 @@
+ <!-- order: 1 -->
+
+# Concepts
+
+## Cronos
+
+Cronos is the name of Crypto.org EVM Chain, also the name of the main module that glues multiple components together for Cronos chain.
+
+## Gas Token
+
+Cronos use CRO as the gas token.
+
+The CROs on the Cronos chain need to be transferred from the Crypto.org chain through an IBC channel, they arrive on the Cronos chain as IBC tokens first, then get converted to the gas token automatically.
+
+### Decimal Places Conversion
+
+The CRO on the Crypto.org chain has 8 decimals, while the Cronos gas token has 18 decimals (to keep compatibility with Ethereum). So a conversion needs to be done:
+
+- When transferring CRO to Cronos chain, the decimal places of the amount are expanded to 18.
+- When transferring CRO from Cronos chain, the amount is truncated to 8 decimals, the remaining part is left in Cronos, so the total value is preserved.
+
+## Native Token
+
+Native token is a token managed by cosmos native bank module, there are several kinds of native tokens in Cronos:
+
+- gas token. used to pay the gas fee.
+- staking token. used to do PoA consensus.
+- IBC token. tokens come from IBC channels.
+- gravity token. tokens come from the gravity bridge.
+
+## CRC20 Token
+
+CRC20 token is Cronos's equivalence of ERC20 token on Ethereum with some extensions, they can be mapped with native tokens and support transfer to/from native tokens, and potentially transfer to/from Ethereum and another cosmos chain through gravity bridge and IBC.
```

Comment:
I think the short description `CRC20 token is Cronos's equivalence of ERC20 token on Ethereum with some extensions` is enough.

We probably want to have more details in another sections for CRC proposal

Reply:
The CRC standard can be described in another place, not necessarily a part of Cronos module spec I think.